pkg/logger/zerolog/zerolog.go

```diff
@@ -0,0 +1,57 @@
+package zerolog
+
+import (
+	"io"
+	"os"
+
+	"github.com/rs/zerolog"
+)
+
+type (
+	// ConsoleWriter is an alias of [zerolog.ConsoleWriter]
+	ConsoleWriter = zerolog.ConsoleWriter
+)
+
+// DefaultLogWriter returns a zerolog.ConsoleWriter set to
+// use os.Stderr which is used if none is given
```

Comment:
This doc string is confusing. I *presume*  it should be:
```go
// DefaultLogWriter returns a zerolog.ConsoleWriter set to
// use os.Stderr as io.Writer because no io.Writer was provided
```